lib/internal/modules/esm/initialize_import_meta.js

```diff
@@ -47,6 +54,26 @@ function initializeImportMeta(meta, context, loader) {
   return meta;
 }
 
+/**
+ * Inspect the module URL and convert it to `dirname` and `filename`
+ * references if it does not point to a network module.
+ * @param {string} url
+ * @returns {{dirname?: string, filename?: string}}
+ */
+function resolveModuleMeta(url) {
+  if (StringPrototypeSlice(url, 0, 7) !== 'file://') {
```

Comment:
```suggestion
  if (StringPrototypeStartsWith(url, 'file://')) {
```

Plus the import at the top.